telegrambot/models.py

```diff
@@ -253,6 +253,10 @@ class PrivilegeScopes(models.TextChoices):
                   "demote administrators that he has promoted, directly or indirectly "
                   "(promoted by administrators that were appointed by the user)",
     )
+    can_superban_members = models.BooleanField(
+        help_text="True, if the administrator can ban a member from all groups of the network. This should be set"
+                  "only for the members of the C.A.N.",
```

Comment:
`This should be setonly for the members of the C.A.N.`
Add a space between `set` and `only`